src/Adapter/Product/Combination/Create/CombinationCreator.php

```diff
@@ -92,7 +92,7 @@ public function createCombinations(ProductId $productId, array $groupedAttribute
         $this->disableSpecificPriceRulesApplication();
 
         $combinationIds = $this->addCombinations($product, $generatedCombinations);
-        $this->updateProductDefaultCombination($productId);
+        $this->combinationRepository->refreshDefaultCombination($productId);
```

Comment:
This one is not useful any more since in the loop that adds combination you use `combinationRepository->create` which already calls `refreshDefaultCombination` internally (at least when no combination is already set as default)

Reply:
Indeed. Actually object model updates default combination after each add/update, so I've removed this completely.